cynthion/python/pyproject.toml

```diff
@@ -34,6 +34,7 @@ dependencies = [
     "tabulate",
     "prompt_toolkit",
     "pygreat @ git+https://github.com/antoinevg/libgreat.git@antoinevg/moondancer#subdirectory=host",
+    "apollo_fpga @ git+https://github.com/greatscottgadgets/apollo.git",  # temporarily pull from git until apollo release
     "apollo_fpga",
```

Comment:
The `apollo_fpga` line should be removed -- although I _am_ curious which of the two setuptools will choose!